### PR TITLE
Huong/test inference

### PR DESCRIPTION
**Pull request by huonglarne**
- Guide to install dependencies for Moreh
- Code to run inference with all available models

### CODE REVIEW

**Comment by huonglarne** [comment]
@yeomni Can you help review?

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
+Install + Run with Moreh 
+ 
+    pip install -e .[all]
+    update-moreh --force --torch 1.10.0
```
Comment (yeomni) [inline README.md]: Let's rather use torch 1.13.1

File: requirements.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@ numpy==1.21.6;python_version<"3.10"
 numpy;python_version=="3.10"
 cython==0.29.28
 scipy>=1.4.0
-torch>=1.7
+torch==1.10.0
```
Comment (yeomni) [inline requirements.txt]: Let's go with 1.13.1

File: README.md [hunk 3]
```diff
@@ -1,4 +1,9 @@
+Install + Run with Moreh
```
Comment (yeomni) [inline README.md]: It also should include conda create/activate/deactivate before installing dependencies.

Comment (huonglarne) [reply README.md]: I added the conda create and activate steps in the latest commit

File: requirements.txt (hunk 2 above)
Comment (huonglarne) [reply requirements.txt]: Done. I tested the models with 1.13.1 and they work fine. I updated the requirements and install commands.

**Review by yeomni — APPROVED** [review]
LGTM!

**Pull request merged**